components/bl-avatar-generator/src/avatar-parts/accessory.js

```diff
@@ -0,0 +1,186 @@
+          id="Frame"/>
+      </g>
+    </g>
+  </g>
+);
+
+const Wayfarers = () => (
+  <g id="Top/_Resources/Wayfarers" fill="none" transform="translate(62.000000, 85.000000)" strokeWidth="1">
+    <defs>
+      <linearGradient x1="50%" y1="0%" x2="50%" y2="100%" id="linearGradient1">
+        <stop stopColor="#FFFFFF" stopOpacity="0.5" offset="0%"/>
+        <stop stopColor="#000000" stopOpacity="0.5" offset="70.5058195%"/>
+      </linearGradient>
+      <path
+        d="M44.9178864,17.5714286 C44.9178864,27.2737857 36.66775,35.1428571 22.9204545,35.1428571 L20.1704091,35.1428571 C6.42311364,35.1428571 0.923022727,27.2708571 0.923022727,17.5714286 L0.923022727,17.5714286 C0.923022727,7.86614286 2.20715909,0 21.4545455,0 L24.3863636,0 C43.63375,0 44.9178864,7.86614286 44.9178864,17.5714286 L44.9178864,17.5714286 Z"
+        id="WayfarersPath1"/>
+      <path
+        d="M106.486068,17.5714286 C106.486068,27.2737857 98.2388636,35.1428571 84.4886364,35.1428571 L81.7385909,35.1428571 C67.9912955,35.1428571 62.4912045,27.2708571 62.4912045,17.5714286 L62.4912045,17.5714286 C62.4912045,7.86614286 63.7753409,0 83.0227273,0 L85.9545455,0 C105.199,0 106.486068,7.86614286 106.486068,17.5714286 L106.486068,17.5714286 Z"
+        id="WayfarersPath2"
+      />
+    </defs>
+    <g id="Wayfarers" transform="translate(7.000000, 7.000000)">
+      <g id="Shades" transform="translate(10.795455, 2.928571)" fillRule="nonzero">
+        <g id="Shade">
+          <use fillOpacity="0.700000048" fill="#000000" fillRule="evenodd" xlinkHref="#WayfarersPath1"/>
+          <use fill="url(#linearGradient1)" fillRule="evenodd" style={{ mixBlendMode: "screen" }} xlinkHref="#WayfarersPath1"/>
+        </g>
+        <g id="Shade">
+          <use fillOpacity="0.700000048" fill="#000000" fillRule="evenodd" xlinkHref="#WayfarersPath2"/>
+          <use fill="url(#linearGradient1)" fillRule="evenodd" style={{ mixBlendMode: "screen" }} xlinkHref="#WayfarersPath2"/>
+        </g>
+      </g>
+      <path
+        d="M33.7159091,41 L30.9658636,41 C17.0778409,41 8.78665909,33.3359286 8.78665909,20.5 C8.78665909,10.127 10.5985227,0 32.25,0 L35.1818182,0 C56.8332955,0 58.6451591,10.127 58.6451591,20.5 C58.6451591,32.5686429 48.3955227,41 33.7159091,41 Z M32.25,5.85421429 C14.6502955,5.85421429 14.6502955,12.3175714 14.6502955,20.5 C14.6502955,27.1800714 17.4795,35.1428571 30.9658636,35.1428571 L33.7159091,35.1428571 C44.9418409,35.1428571 52.7815227,29.1217143 52.7815227,20.5 C52.7815227,12.3175714 52.7815227,5.85421429 35.1818182,5.85421429 L32.25,5.85421429 Z"
+        id="Left"
+        fill="#252C2F"
+        fillRule="nonzero"/>
```

Comment:
Here and everywhere in this file and all others files maximum allowed line length is 120 and closing tag at the new line.